extensions/lorem-ipsum/package.json

```diff
@@ -6,20 +6,53 @@
   "icon": "paragraph-icon.png",
   "author": "AntonNiklasson",
   "license": "MIT",
+  "contributors": [
+    "GastroGeek"
+  ],
+  "categories": [
+    "Data",
+    "Design Tools",
```

Comment:
Does this really fit in `Design Tools`, `Web`, and `Developer Tools`? Thoughts @pernielsentikaer ?

Reply:
Just my 2c... **yes**, since as a developer and/or designer I often need to generate lots of lorem text as placeholder text... 🤷 both of which often relate to 'web'...

Reply:
I think we can keep it as a Design Tool too 😊

Reply:
Great, makes a lot of sense! ✨